users/templates/registration/send_invitation.html

```diff
@@ -0,0 +1,57 @@
+{% extends 'base.html' %}
```

Comment:
Use: {% extends 'user_profile/base_user_profile.html' %},
this way the view will appear in the same place the other stuff from user menu

Reply:
Done